Source/CLI/Help.cpp

```diff
@@ -11,18 +11,53 @@
 //---------------------------------------------------------------------------
 
 //---------------------------------------------------------------------------
+#include "Common/Core.h"
 #include "Help.h"
 #include "Config.h"
 //---------------------------------------------------------------------------
 
 //***************************************************************************
 //
 //***************************************************************************
+//---------------------------------------------------------------------------
+String Program_Name(const String &Set)

```

Comment:
Prefer to use a setter and a getter, not both in the same function (if Set is not empty, you don't need the returning String)